src/SFA.DAS.ApplyService.Web/Services/RoatpTaskListWorkflowService.cs

```diff
@@ -36,7 +37,25 @@ public  string SectionStatus(IEnumerable<ApplicationSequence> applicationSequenc
 
             var questionsCompleted = SectionHasCompletedQuestions(section);
 
-            return SectionText(questionsCompleted, section.SectionCompleted, sequential);
+            // NOTES: MFCMFC The SectionText() method can be removed once the QnA json and process conforms to writing details
+            // into the QnA, and has been fixed.  Current areas affected is all of sequence 1
+            // In addition, there is probably a case for removing all calls and reads to the ApplyRepository 'Completed' calls
+            // (MarkSectionAsCompleted, IsSectionCompleted,RemoveSectionCompleted), and the table 'ApplicationWorkflow' may be dropped
+            // I will need to double check there are no other uses for this endpoint before doing that
+            var sectionCompleteBasedOnPagesActiveAndComplete = GetSectionText(questionsCompleted, section,sequential);
+            var sectionCompleteBasedOnDatabaseSettingOfIsComplete = SectionText(questionsCompleted, section.SectionCompleted, sequential);
+
+            var sectionText = sectionCompleteBasedOnPagesActiveAndComplete;
+
+            if (sectionCompleteBasedOnDatabaseSettingOfIsComplete != sectionCompleteBasedOnPagesActiveAndComplete)
+            {
+                if (sequenceId == 1)
```

Comment:
If this becomes an enduring solution rather than a temporary one then we should use the RoatpWorkflowSequenceIds.YourOrganisation constant